content/blogposts/2020/series-c-with-sequoia.md

```diff
@@ -9,7 +9,7 @@ tags: [
 ]
 slug: series-c-with-sequoia
 heroImage: /blog/sourcegraph-social-img.png
-socialImage: https://about.sourcegraph.com/blog/sourcegraph-social-img.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/sourcegraph-social-image.png
```

Comment:
Since all the markdowns get parsed + rendered through `<Layout />`, an absent `socialImage` should set this image as the default & this line can be omitted. Do you get those results?

Reply:
Agreed 👍 These three instances should be removed now.